src/sdl2_compat.c

```diff
@@ -8163,7 +8175,9 @@ SDL_JoystickPathForIndex(int idx)
 SDL_DECLSPEC SDL2_bool SDLCALL
 SDL_JoystickHasLED(SDL_Joystick *joystick)
 {
-    return SDL3_GetBooleanProperty(SDL3_GetJoystickProperties(joystick), SDL_PROP_JOYSTICK_CAP_RGB_LED_BOOLEAN, false) ? SDL2_TRUE : SDL2_FALSE;
+    bool has_led = SDL3_GetBooleanProperty(SDL3_GetJoystickProperties(joystick), SDL_PROP_JOYSTICK_CAP_RGB_LED_BOOLEAN, false) ||
+                   SDL3_GetBooleanProperty(SDL3_GetJoystickProperties(joystick), SDL_PROP_JOYSTICK_CAP_MONO_LED_BOOLEAN, false);
+    return has_led ? SDL2_TRUE : SDL2_FALSE;
```

Comment:
SDL_JoystickHasLED() in SDL2 only returned true for RGB LED support. This change should be dropped.

Reply:
retracted the commit changing JoystickHasLED